storage/src/meta/chunk_info_v2.rs

```diff
@@ -46,17 +47,41 @@ impl BlobChunkInfoV2Ondisk {
         }
     }
 
+    pub(crate) fn set_batch(&mut self, batch: bool) {
+        if batch {
+            self.uncomp_info |= u64::to_le(CHUNK_V2_FLAG_BATCH);
+        } else {
+            self.uncomp_info &= u64::to_le(!CHUNK_V2_FLAG_BATCH);
+        }
+    }
+
     pub(crate) fn set_data(&mut self, data: u64) {
         self.data = u64::to_le(data);
     }
 
     pub(crate) fn set_zran_index(&mut self, index: u32) {
+        assert!(self.is_zran());
         let mut data = u64::from_le(self.data) & !0xffff_ffff_0000_0000;
         data |= (index as u64) << 32;
         self.data = u64::to_le(data);
     }
 
     pub(crate) fn set_zran_offset(&mut self, offset: u32) {
+        assert!(self.is_zran());
+        let mut data = u64::from_le(self.data) & !0x0000_0000_ffff_ffff;
```

Comment:
Could you please help to simplify `!0x0000_0000_ffff_ffff` to `0xffff_ffff_0000_0000`?

Reply:
Changed.